src/Angor/Avalonia/Angor.Contexts.Funding/Investor/Domain/InvestmentRecord.cs

```diff
@@ -10,4 +10,13 @@ public class InvestmentRecord
     
     public DateTime? RequestEventTime { get; set; }
     public string? RequestEventId { get; set; }
+    
+    public string? RecoveryTransactionId { get; set; }
+    public string? RecoveryTransactionHex { get; set; }
```

Comment:
Why do we need the hex of all these transactions? we only keep the hex of the investment because we need the founder to sign on it as well